.github/workflows/cli.yml

```diff
@@ -12,88 +12,118 @@ concurrency:
 
 jobs:
   build:
-    name: Build with Node ${{ matrix.node }}
-    runs-on: ubuntu-latest
+    name: Build ${{ matrix.package }} with Node ${{ matrix.node }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         node: ['12', '14', '16']
+        package: ['cli']
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
```

Comment:
I think we should make an effort to support the three platforms. I've seen many frameworks building and testing only for Unix-based OSs and leaving Windows as an after-thought. I think we can do better :)